jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/roles/CaseRolesViewImpl.java

```diff
@@ -65,87 +70,113 @@
     @DataField("pagination")
     private PaginationViewImpl pagination;
 
-    List allElementsList = new ArrayList();
-
-    private Command userAddCommand;
+    List <CaseRoleItemView> allFilteredElements = new ArrayList();
+    List <CaseRoleItemView> allUnfilteredElements = new ArrayList();
 
     @Inject
     private ManagedInstance<CaseRoleItemView> provider;
 
-    @Override
-    public void init(final CaseRolesPresenter presenter) {
-
-    }
+    @Inject
+    private TranslationService translationService;
 
     @Override
-    public void removeAllRoles() {
-        removeAllChildren(roles);
-        allElementsList = new ArrayList();
+    public void init(final CaseRolesPresenter presenter) {
     }
 
-    @Override
-    public void enableNewRoleAssignments() {
-        removeCSSClass(userAddButton, "hidden");
-    }
+    @PostConstruct
+    public void init() {
+        stream(CaseRolesAssignmentFilterBy.values())
```

Comment:
_stream(CaseRolesAssignmentFilterBy.values()).forEach_ seems all that you need in here to iterate over all elements, or even a simple _for_ would be enough.